### PR TITLE
2017-18 Schedule

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: football.json [hunk 1]
```diff
@@ -1,98 +1,98 @@
...
-	{
-		"id":"12",
-		"opponent":"W. Virginia Mountaineers",
-		"location":"Away",
-		"date":"2016-12-03",
-		"kickoff":"TBA",
-		"televised":"FS1"
-	}
+    {
+        "id":"2025931",
```
Comment (bradsbrown) [inline football.json:3]: Out of curiosity, are you using some API to get this data? I notice our IDs went from a game count to...something else?

**Comment by bradsbrown** [comment]
@arontx back to you with a quick question

**Comment by ghost** [comment]
I'm not using an API; that's just what was in the source on BU's schedule page. We're not using the ID for anything so it doesn't matter one way or the other.

**Comment by bradsbrown** [comment]
Yeah, just idle curiosity. Looks good to me!

**Pull request merged**